pkg/metrics/metrics.go

```diff
@@ -32,8 +32,12 @@ const (
 	backupSuccessCount          = "backup_success_total"
 	backupFailureCount          = "backup_failure_total"
 	backupDurationSeconds       = "backup_duration_seconds"
+	restoreAttemptCount         = "restore_attempt_total"
+	restoreSuccessCount         = "restore_sucess_total"
+	restoreFailureCount         = "restore_failure_count"
 
-	scheduleLabel = "schedule"
+	scheduleLabel   = "schedule"
+	backupNameLabel = "backupName"
```

Comment:
Although I don't think we want to group restore metrics by backup. That means we'll get unique restore attempt/success/failure metrics per backup. It might be better to group by schedule, like we do for backups.

Reply:
I was wondering about that - do we have the schedule information available on restore?

Reply:
It's available as a label in the backup (which we have).